.gitlab-ci.yml

```diff
@@ -235,6 +235,7 @@ migrate:
   script:
     - *bundle_install
     - bundle exec rake db:create db:migrate --trace
+    - bundle exec rake db:check_for_sensitive_columns
```

Comment:
what if we added this code as a spec instead? `spec/db/schema_spec.rb` and had it do the same check? easier to see output in tests or re-run